FoodDeliveryUI/FoodDeliveryUI/FoodDeliveryUI.Shared/MainPage.xaml

```diff
@@ -0,0 +1,80 @@
+            <Image Source="book"/>
+        </Button>
+
+        <!-- 2. Menu to Eat-->
+
+        <!-- Gretting label -->
+        <TextBlock Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="1" Text="Hello, Rachel!" FontSize="30" Foreground="White" FontWeight="Bold" Margin="0,20,0,0" />
+        <!-- Welcome Label-->
+        <TextBlock Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="2" Text="Wanna eat tonight?" FontSize="15" Foreground="White"/>
+        <!-- Menu list-->
+        <ListView Grid.Row="3" Grid.Column="0" Grid.ColumnSpan="3" Margin="0,20,0,0" ItemsSource="{x:Bind food}">
+            <ListView.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <StackPanel Orientation="Horizontal"/>
+                </ItemsPanelTemplate>
+            </ListView.ItemsPanel>
+            <ListView.ItemTemplate>
+                <DataTemplate x:DataType="local:Food">
+                    <Frame CornerRadius="15" BorderBrush="#1a204e" BorderThickness="1" Height="200" Width="150" HorizontalAlignment="Left" Margin="-5,0" >
+                        <Grid Padding="10" RowDefinitions="*,Auto,Auto">
+                            <Image Grid.Row="0" Source="{x:Bind Picture}" Height="100" Width="100"/>
+                            <TextBlock Grid.Row="1" Text="{x:Bind Name}" FontWeight="Bold" Foreground="White" HorizontalAlignment="Center"/>
+                            <TextBlock Grid.Row="2" Text="{x:Bind Description}" Foreground="White" HorizontalAlignment="Center"/>
+                        </Grid>
+                    </Frame>
+                </DataTemplate>
+            </ListView.ItemTemplate>
+        </ListView>
+        <!-- Horizontal Frame-->
+        <Frame Grid.Row="4" Grid.Column="0" Grid.ColumnSpan="3" CornerRadius="15" BorderBrush="#1a204e" BorderThickness="1" HorizontalAlignment="Stretch"  Height="100" Margin="0,10">
+            <Grid Padding="10" RowDefinitions="*,*" ColumnDefinitions="*,Auto">
+                <TextBlock Grid.Row="0" Grid.Column="0" Text="See All Categories" FontWeight="Bold" Foreground="White" VerticalAlignment="Bottom" />
+                <TextBlock Grid.Row="1" Grid.Column="0" Text="100 Categories" Foreground="White" />
+                <Image Grid.Row="0" Grid.Column="1" Grid.RowSpan="2" Source="food3" Height="100" Width="100" VerticalAlignment="Bottom" HorizontalAlignment="Right" />
+            </Grid>
+        </Frame>
+
+
+        <!-- 3. Recommended-->
+        
+        <!-- Title-->
+        <TextBlock Grid.Row="5" Grid.Column="0" Text="RECOMMENDED" FontWeight="Bold" Foreground="White" Margin="0,20"/>
+        <!-- Recommended menu-->
+        <ListView Grid.Row="6" Grid.Column="0" Grid.ColumnSpan="3"  ItemsSource="{x:Bind recommendations}">
+            <ListView.ItemTemplate>
+                <DataTemplate x:DataType="local:Recommendation">
+                    <Grid RowDefinitions="Auto,Auto,Auto" ColumnDefinitions="Auto,*" Padding="0,0,0,15">

```

Comment:
the selection looks off-centered:
![image](https://user-images.githubusercontent.com/2359550/201988714-81025b6e-5037-4815-aa3e-af6fb65ff82c.png)

consider using a even or more or less even top-bottom padding like `0,8` or `0,7,0,8`.
or, you can disable selection all together if that is not intended with `SelectionMode=None`.